<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManager</class>
 <widget class="QWidget" name="OrderManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="0,0,4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTreeView" name="orderTreeView"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget_4">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="m_customerAddTab_5">
        <attribute name="title">
         <string>입력</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_50">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_52">
             <item>
              <widget class="QLabel" name="label_46">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>orderID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="orderInputIdText">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_53">
             <item>
              <widget class="QLabel" name="label_47">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>clientID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="orderInputClientIdText"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_56">
             <item>
              <widget class="QLabel" name="label_51">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>clientName</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="orderInputClientNameText"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_54">
             <item>
              <widget class="QLabel" name="label_48">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>productId</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="orderInputProductIdText"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_55">
             <item>
              <widget class="QLabel" name="label_49">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>date</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="orderInputDateText"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_66">
             <item>
              <widget class="QLabel" name="label_60">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>orderPrice</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="orderInputOrderPriceText"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_63">
             <item>
              <widget class="QLabel" name="label_57">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>orderStock</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="orderInputOrderStockText"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_13">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <item>
              <widget class="QPushButton" name="orderInputConfirmButton">
               <property name="text">
                <string>confirm</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>21</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="orderModifyButton">
               <property name="text">
                <string>modify</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_customerSerchTab_5">
        <attribute name="title">
         <string>검색</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_50">
           <property name="text">
            <string>searchElements</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="orderComboBox">
           <item>
            <property name="text">
             <string>orderID</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>clientID</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>clientName</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>productId</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>productName</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>date</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>orderPrice</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>orderStock</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_14">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_58">
           <item>
            <widget class="QLabel" name="label_52">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Elements</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="orderSearchText">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="orderResetButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/images/eraser2.png</normaloff>:/images/eraser2.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_15">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>167</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="orderSearchButton">
           <property name="text">
            <string>search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
